components/widgets/forest-change/glad-alerts-simple/selectors.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable prefer-destructuring */
+import { createSelector, createStructuredSelector } from 'reselect';
+import isEmpty from 'lodash/isEmpty';
+import { formatNumber } from 'utils/format';
+
+import moment from 'moment';
+
+// get list data
+const selectAlerts = (state) => state.data && state.data.alerts;
+const selectColors = (state) => state.colors;
+const selectSentences = (state) => state.sentence;
+const getIndicator = (state) => state.indicator || null;
+const getSettings = (state) => state.settings || null;
+const getLocationName = (state) => state.locationLabel;
+
+export const parseData = createSelector([selectAlerts], (data) => {
+  if (!data || isEmpty(data)) return null;
+  const otherAlertsData = data.filter((d) => d.confirmed === false);
+  const confimedAlertsData = data.filter((d) => d.confirmed === true);
+
+  const otherAlerts = otherAlertsData.length ? otherAlertsData[0].alerts : 0;
+  const highConfidenceAlerts = confimedAlertsData.length
+    ? confimedAlertsData[0].alerts
+    : 0;
+
+  const totalAlerts = otherAlerts + highConfidenceAlerts;
+
+  return {
+    totalAlertCount: totalAlerts,
+    otherAlertCount: otherAlerts,
+    otherAlertPercentage: (100 * otherAlerts) / totalAlerts,
+    highConfidenceAlertCount: highConfidenceAlerts,
+    highConfidenceAlertPercentage: (100 * highConfidenceAlerts) / totalAlerts,
+  };
+});
+
+export const parseConfig = createSelector(
+  [parseData, selectColors, getIndicator],
+  (data, colors, indicator) => {
+    if (isEmpty(data)) return null;
+
+    const {
+      otherAlertCount,
+      otherAlertPercentage,
+      highConfidenceAlertCount,
+      highConfidenceAlertPercentage,
+    } = data;
+    const alertsLabel = indicator
+      ? `Other alerts in ${indicator.label}`
+      : 'Other alerts';
+    const highConfidenceAlertsLabel = indicator
+      ? `High confidence alerts in ${indicator.label}`
+      : 'High confidence alerts';
+
+    const highConfidenceColour = colors.main;
+    const otherColour = colors.gladOther; // hslShift(mainColour)
+    const parsedData = [
+      {
+        label: highConfidenceAlertsLabel,
+        value: highConfidenceAlertCount,
+        color: highConfidenceColour,
+        percentage: highConfidenceAlertPercentage,
+        unit: 'counts',
+      },
+      {
+        label: alertsLabel,
+        value: otherAlertCount,
+        color: otherColour,
+        percentage: otherAlertPercentage,
+        unit: 'counts',
+      },
+    ];
+    // if (indicator) {
```

Comment:
if this should stay in the code base, add a comment on why its commented out / might be needed later

Reply:
It can go - no further intersections shown in the chart